even_odd.rb

```diff
@@ -8,3 +8,4 @@ def find_even_odd(number)
 puts find_even_odd(0)
 puts find_even_odd(23)
 puts find_even_odd(100)
+
```

Comment:
Not to be used predefined method, search for custom method , the approach you have used